deploy_hub.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+TAG="${TRAVIS_TAG:-latest}"
```

Comment:
`latest` is confusing. It sounds like the latest "release", but it is actually not release. I suggest just to use `TRAVIS_BRANCH`; a user should know that something from Mastodon named `master` is unstable.

Reply:
I'm just trying to replicate current behavior. Currently `latest` points at the tip of master (although it hasn't been built in a while due to the bug this PR is trying to solve), as far as I can tell, hence the same behavior in here.

Reply:
`latest` has a tradition of being bleeding edge in any case - https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375

Reply:
> I'm just trying to replicate current behavior. Currently `latest` points at the tip of master (although it hasn't been built in a while due to the bug this PR is trying to solve), as far as I can tell, hence the same behavior in here.

I do not think there is any strict rule yet. It is a good chance to consider about it well.

> If latest should be bleeding edge (I'd be in favor of it) or not isn't for this PR to decide.

It is if there is no rule yet.

> `latest` has a tradition of being bleeding edge in any case

Bleeding edge "release", maybe? Indeed many Docker images are just for releases and it is hard to find examples of CI.